packages/web-runtime/src/helpers/additionalTranslations.ts

```diff
@@ -29,5 +30,19 @@ export const additionalTranslations = {
   shareRoleLabelViewer: $gettext('Can view'),
   shareRoleLabelEditor: $gettext('Can edit'),
   shareRoleLabelUploader: $gettext('Can upload'),
-  shareRoleLabelManager: $gettext('Can manage')
+  shareRoleLabelManager: $gettext('Can manage'),
+  knownSharePermissions: [
+    $gettext('libre.graph/driveItem/upload/create'),
+    $gettext('libre.graph/driveItem/permissions/create'),
+    $gettext('libre.graph/driveItem/children/create'),
+    $gettext('libre.graph/driveItem/basic/read'),
+    $gettext('libre.graph/driveItem/path/read'),
+    $gettext('libre.graph/driveItem/quota/read'),
+    $gettext('libre.graph/driveItem/content/read'),
+    $gettext('libre.graph/driveItem/children/read'),
+    $gettext('libre.graph/driveItem/deleted/read'),
+    $gettext('libre.graph/driveItem/path/update'),
+    $gettext('libre.graph/driveItem/deleted/update'),
+    $gettext('libre.graph/driveItem/standard/delete')
```

Comment:
@tbsbdr This is how the permissions are currently listed if a share/space role has been disabled on the server. Is this okay like this (for now), or should we replace it with more "meaningful" texts?

Reply:
you don't have a screenshot, do you?

Reply:
Looks like this for the "Secure view" role e.g. (the margins of our contextual helper are a bit off):

<img width="426" alt="image" src="https://github.com/user-attachments/assets/4aaf5139-1cb2-416e-a49c-faf9d884c36f">

Reply:
As discussed: we leave these permissions untranslated (for now). As soon as we have a doc link, we'll add it to the context helper so users can make sense of it (see https://github.com/owncloud/web/issues/11392).